utils/config.py

```diff
@@ -0,0 +1,40 @@
+from ast import literal_eval
+import json
+from os import path
+
+class Config():
+
+
+    def __init__(self):
+        self.conf_file = 'config.json'
```

Comment:
This is merely a suggestion, but since you already have a `Config` class, why not make it more extensible? A good example would be to make it load any `json` file
```suggestion
    def __init__(self, filename='config.json'):
        self.conf_file = filename
```
Example how this can be utilized in `bot.py`:
```py
self.blocked_users = Config('blocked_users.json')
```